### ISSUE TITLE
3rd Person View

### ISSUE THREAD

**Issue by ghost** [Overv/WebCraft#48]
I deeply apologize for my issue coming across that way. 
I did not mean i did not know about js, I meant I did not know about WebGL affiliated js.
I have made things in three.js, but other than that nothing in WebGL.
When I asked to open a codepen page I meant that more than 1 person could edit; not that contained all the code that someone worked on for me.
I realize now how someone could have been greatly offended and I thank you for being kind enough to point that out.  That was all I needed to know when you pointed me to the function.
I will close the issue after you read this.
Again thank you for pointing out my mistake and I apologize for offending you and anyone else who read this and was offended.

**Comment by cymno** [comment]
It's considered rude to ask for help (= work) if you don't have anything to show that you already put in at least some effort yourself, it makes it look like you don't appreciate other peoples time.

That said, if you want to use this project to learn a bit of programming, here are some steps you should take:

- Learn about git and github (e.g. from youtube tutorials, blog posts etc). jsfiddle and the likes are great for quick prototyping, but for larger projects precise tracking of changes is necessary. Try forking this repo and making a small commit yourself

- Get the project to run locally on your machine. You can find some precise instructions in some of the older issues on this repo.

- Get a rough overview of which code is run in what order when you start the project. You can simply follow function calls through the several program files.

- Find the location where a simple value like the player movement speed or jump height is defined, change it and observe if it results in a change in-game

There is no 3rd person view currently in the game, but there is some basic animation (arm leg head movement). For a simple 3rd person view find the setCamera method in the multi/singlepl.html files and move its input coordinates backwards.

When you have made some progress to show, and have a question on a *precise* problem you couldn't solve after trying for some hours, concerning *this* specific project, feel free to ask

**Comment by cymno** [comment]
Oh wow, I came off pretty rude myself, didn't mean to scare you off! Don't worry about making mistakes, just try things out and you'll learn from them and  get better all the time. Better to piss off some peoplethan giving up. Don't say "I have cool idea, code it for me!" say "I want to program this, how to do it? I tried x and didn't help", and people will go to lots of efforts to help you, because helping is fun.

Btw I didn't get notified of your answer until now because it was done as an edit.
